superset-frontend/src/components/CronPicker/CronPicker.tsx

```diff
@@ -44,7 +44,7 @@ export const LOCALE: Locale = {
   prefixMonths: t('in'),
   prefixMonthDays: t('on'),
   prefixWeekDays: t('on'),
-  prefixWeekDaysForMonthAndYearPeriod: t('and'),
+  prefixWeekDaysForMonthAndYearPeriod: t('or'),
```

Comment:
This one was from a suggestion in a comment on [this ticket](https://app.shortcut.com/preset/story/77442/combine-on-and-and-to-the-same-line-in-weekly-schedule). Zef thought it makes more sense because the cron fires if either are true like an OR statement, but, it can be troubling if your reading it as natural language. Not sure what is more appropriate.

Reply:
[This answer](https://unix.stackexchange.com/questions/602328/are-the-day-of-month-and-day-of-week-crontab-fields-mutually-exclusive#:~:text=Finally%2C%20if%20either%20the%20month,of%20week%2C%20shall%20be%20matched.) explains the crontab behavior. Semantically it seems the 'or' is more appropriate. It is a little clumsy when someone only wants use of the 'day of week' setting as they might suspect leaving the 'day of month' setting unchanged would cause it to run 'every day of the month'.